src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -825,6 +825,36 @@ def generateBaseMesh(region, options):
                 bodyMeshGroup.addElement(element)
                 elementIdentifier += 1
 
+        # create 2d mesh groups
+        fm.defineAllFaces()
+        bodyGroup.addSubelements()
+        neckGroup.addSubelements()
+        mesh2d = fm.findMeshByDimension(2)
+        is_exterior = fm.createFieldIsExterior()
+        is_exterior_face_xi3_1 = fm.createFieldAnd(is_exterior, fm.createFieldIsOnFace(Element.FACE_TYPE_XI3_1))
+        is_interior = fm.createFieldIsExterior()
```

Comment:
Remove is_interior and just use is_exterior. Exterior here means on a surface or edge of an element without an adjacent element on any side, so it applies on the inside and outside of a walled body such as the bladder.